libs/stack/stack-ui/src/components/fields/RadioGroup/radio-group.stories.jsx

```
@@ -0,0 +1,51 @@
+}
+
+export const Disabled = {
+  render: Template.bind({}),
+  name: 'Disabled',
+
+  args: {
+    isDisabled: true,
+  },
+}
+
+export const Error = {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)